airgun/entities/contentview.py

```diff
@@ -33,6 +33,10 @@ def update(self, entity_name, values):
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
         return view.fill(values)
 
+    def add_yum_repo(self, entity_name, repo_name):
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
```

Comment:
Hm... Shouldn't we have navigate step for navigating 'Yum content' tab? Won't we have difficulties with understanding where we are when we introduce corresponding logic later?

Reply:
can you provide more details to that question? I am using the same idea that we used for associate product in activation key. Do we have changes in approach?

Reply:
Yes, i know we haven't thought about that before, it's not a blocker for current PR. I'm just wondering whether navigator should be aware of all the tabs. I can see cfme does have tabs registered as nav steps, at least some of them, which is probably allowing to figure out where user currently is and whether navigation is needed. On the other hand, all the tabs have a logic like "if user is not here - click the tab first", so maybe that's redundant. @mfalesni any details?